source-code/ide-extension/src/main.ts

```diff
@@ -55,33 +57,39 @@ async function main(args: { context: vscode.ExtensionContext }): Promise<void> {
     options: potentialConfigFileUris.map((uri) => uri.path),
     to: activeTextEditor.document.uri.path,
   });
-  const configModule: InlangConfig = (await import(closestConfigPath)).config;
+
+  // Change current working directory to configuration file directory, so relative paths to the languages files work.
+  // Otherwise the current working directory path is the directory where the node binary resides.
+  process.chdir(dirname(closestConfigPath));
+
+  // TODO: find better fs (vscode.workspace.fs)
+  const $import = initialize$import({ fs: fs.promises as any, fetch });
+  const configModule: InlangConfig = await (await import(closestConfigPath)).defineConfig({ $fs: fs.promises, $import });
```

Comment:
I would split this line of code into two lines: 

```ts
const module = await import(closestConfigPath)
const config = await module.defineConfig({ $fs: fs.promises, $import })
```

For minor change suggestions like this one, should I go ahead and commit an improvement directly?

Reply:
I'll change this one, but I'm totally fine if you commit into my branches :)